### ISSUE TITLE
推荐的推理步数是多少？What is the recommended number of reasoning steps?

### ISSUE THREAD

**Issue by smthemex** [WillWu111/FreeSwim#2]
The result of using 25 steps is not very good...
if strength ==0.7 ,the object will change

<img width="1521" height="776" alt="Image" src="https://github.com/user-attachments/assets/be22f58c-1893-40ef-9b8a-e9dadaed0dc9" />

and when use torch2.8 cu12.8 3090 24G  got OOM  just infer 832*480--> 1280*720
if move to cpu , the program crashes directly, it should have caused Ram OOM
init_mask_flex(
        num_frames=1 + (num_frames - 1) // 4, 
        height=width // 16, 
        width=height // 16,
        d_h=base_height // 16 // 2, 
        d_w=base_width // 16 // 2, 
        device='cuda' if torch.version.__version__.startswith("2.7.1") else 'cpu',
        # if you use other torch version, the memory usage would make a big difference, 
        # but you can specify the device to cpu to avoid this problem,
        # which would make the speed slower for around 5s
    )

**Comment by WillWu111** [comment]
1. In our experiments, the recommended inference steps is 50, while the recommended strength is 0.7.
2. So please change the torch version to 2.7.1 to efficiently get the attention mask.

**Comment by smthemex** [comment]
Thanks for your reply. 
When the strength is 0.7, the character in the reference video has changed. Is this a known problem with the method or is it due to fewer steps?

**Comment by WillWu111** [comment]
Thanks for your attention.
In our method, the difference that our method can make is actually **not a problem**. However, the ability of **adding the fine-grained visual details** based on the specific prompts in a training-free way, as you can see on the woman's face and finger, is a biggest difference from those **low-level restoration** models. So, the more **specific** prompts can make more detailed results.
